XBeeModuleSim.py

```diff
@@ -65,45 +65,22 @@ def nop_callback(data):
         self.ground_callback = nop_callback
         """Is called whenever data recieved from the rocket needs to be sent to the rest of the ground station code. The callback should be thread safe."""
 
-        self._running_lock = Lock()
-        self._running = True
-
         # Queues are IO bound.
-        self._rocket_rx_thread = Thread(
-            target=self._run_rocket_rx, name="xbee_sim_rocket_rx"
-        )
+        self._rocket_rx_thread = Thread(target=self._run_rocket_rx, name="xbee_sim_rocket_rx", daemon=True)
 
         self._rocket_rx_thread.start()
 
     def shutdown(self):
```

Comment:
If you're going to make this a no-op you may as well just delete it entirely and remove any references to it (one in SimConnection in shutDown, one in `teardown_method` in the test file.

Reply:
Other than that LGTM.